client/src/Components/Button/StyledAddProspectButton.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Button, withStyles } from "@material-ui/core";
+
+const styles = {
+  root: {
+    backgroundImage: "linear-gradient(to right, #2AA897, #4FBE75)",
+    borderRadius: 7,
+    color: "#FFFFFF",
+    fontWeight: "thin",
+    fontSize: ".8rem",
+    height: 54,
+    width: 180
+  }
+};
+
+const StyledAddProspectButton = props => {
```

Comment:
I liked how you created a specific styled button - I feel like this name though is really specific though, so it's not that reusable - I'd generalize the name here

Also, I wanted to double check with @Disquietude , but the folder structure, it should be lowercase Button right? We only capitalize the folder names in src